.github/workflows/mentions-workflow.yml

```diff
@@ -1,17 +1,23 @@
 name: Mentions workflow
+
 on:
-  #push:
-  #  branches:
-  #    - main
+  issues:
+    types:
+      - opened
```

Comment:
```suggestion
      - closed
```

Reply:
No, actually my idea was like.

_**Process**_
1. User creates an issue.
2. Github workflow got triggered.
    1. Workflow got the username of the account which created the issue
    2. It updated the `MENTIONS.md`
    3. It renders new `README.md`
    4. Then lastly we close the issue, like so
        ```yml
        - name: "Close this issue"
          if: ${{ always() }}
          run: gh issue close $ISSUE --comment "Updated your name on this repo's README.md file!! 🥳🥳"
          env:
            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
            ISSUE: ${{ github.event.issue.html_url }}
        ```

Reply:
Nice demo !

I would say Yes and No !

- The fact that anybody open an issue contribute to the main repository is not appropriate and for me this should be validated somewhere manually. An idea could be the workflow create an automatic Pull Request that have to be manually approve.
- Another thing is that `https://raw.githubusercontent.com/matyo91/matyo91/main/MENTIONS.md` sometimes do not match always with Github username, but this could be the default behaviour for my case.
This is why the issue template provide and URL. May be this can be automatically retrieved by passing [`github.event.issue.body`](https://docs.github.com/en/rest/issues/events) var (first idea it came to me now).

We can continue argue, if you are willing provide more changes.
As your PR go, it's ok for me to merge it as it.
Else I will look how this go with your work, as you get me new ideas and later provide my own custom automation implementation.

Reply:
we can give it a shot!
if worked congratulations
else will go back to last commit?